Snowflake.Data.Tests/UnitTests/Configuration/EasyLoggingConfigFinderTest.cs

```diff
@@ -126,6 +126,21 @@ public void TestThatReturnsNullIfNoWayOfGettingTheFile()
             Assert.IsNull(filePath);
         }
 
+        [Test]
+        public void TestThatReturnsNullIfDirectoryBasedSearchFailsWithUnexpectedError()
+        {
+            // arrange
+            t_environmentOperations
+                .Setup(e => e.GetFolderPath(Environment.SpecialFolder.UserProfile))
+                .Throws(() => new Exception("No home directory"));
+            
+            // act
+            var filePath = t_finder.FindConfigFilePath(null);
```

Comment:
Are we sure the exception has been thrown?
Maybe add `t_environmentOperations.VerifyNoOtherCalls()`

Reply:
done